scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -2066,4 +2066,33 @@ public void endActivity(Body body) {
             throw new SignalApiException("Could not add signal for the job " + jobId, e);
         }
     }
+
+    @Override
+    @ImmediateService
+    public Map<String, Map<String, Boolean>> checkJobsPermissionMethods(List<String> jobIds, List<String> methods)
+            throws NotConnectedException, UnknownJobException {
+        String currentUser = frontendState.getCurrentUser();
+        if (methods == null || jobIds == null) {
+            return Collections.emptyMap();
+        }
+
+        Map<String, Map<String, Boolean>> answer = new HashMap<>(jobIds.size());
+        for (String jobId : jobIds) {
+            Map<String, Boolean> methodsPermissions = new HashMap<>(methods.size());
+            for (String method : methods) {
+                boolean hasPermission = true;
+                try {
+                    frontendState.checkPermissions(method,
+                                                   frontendState.getIdentifiedJob(JobIdImpl.makeJobId(jobId)),
+                                                   SchedulerFrontendState.getErrorMessageForMethodPermission(method));
+                } catch (PermissionException e) {
+                    hasPermission = false;
+                }
+                logger.info("Request permission to " + method + " for " + jobId + " received from " + currentUser);
```

Comment:
Please remove this log entry as it will be constantly called by the scheduler portal, resulting in spamming of messages inside Scheduler.log